numba/targets/arraymath.py

```diff
@@ -2520,6 +2520,15 @@ def np_corrcoef_impl_single_variable(x, y=None, rowvar=True):
 #----------------------------------------------------------------------------
 # Element-wise computations
 
+@overload(np.argwhere)
+def np_argwhere(a):
+
+    def impl(a):
```

Comment:
I'm not sure that Numba will support all types for the argument `a`, what happens if `a` is a string?

This guide should help describe the sort of checks that should take place http://numba.pydata.org/numba-doc/latest/extending/overloading-guide.html